From: John Zupin <johnz@lunarg.com>
Date: Thu, 19 Dec 2024 14:41:48 -0700
Subject: patch: update install path of .cmake files

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b6f68b4..4edf3cd 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -393,5 +393,5 @@ install(
     FILES
         "${PROJECT_BINARY_DIR}/${PROJECT_NAME}Config.cmake"
         "${PROJECT_BINARY_DIR}/${PROJECT_NAME}ConfigVersion.cmake"
-    DESTINATION cmake
+    DESTINATION lib/cmake
 )
