From: John Zupin <johnz@lunarg.com>
Date: Wed, 29 Jan 2025 16:09:46 -0700
Subject: patch: update include path to include/slang

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1e5d1d1..31f8807 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -429,8 +429,8 @@ install(
     PATTERN ".*" EXCLUDE
 )
 install(
-    DIRECTORY "${slang_SOURCE_DIR}/include"
-    DESTINATION .
+    DIRECTORY "${slang_SOURCE_DIR}/include/"
+    DESTINATION include/slang
     PATTERN ".*" EXCLUDE
 )
 include(CPack)
